episodes/introduction.md

```diff
@@ -1,28 +1,213 @@
+another idea instead.
+
+As you prepare for your qualifying exams, you're interested in picking up on their
+work and developing it further. While they give you the go-ahead over email, they let
+you know that they're travelling for field work for the next six months and won't be
+able to respond to further questions - the documents in the drive will be your only
+source of information going forward.
+
+:::::::: challenge
+
+#### Introspection: replication
+
+Think of a time you've had to replicate someone else's research results, either as a class assignment or by inheriting a project someone else (maybe past-you) started. What was hard about it? What was easy? What changes could the original author have made to ease your way?
+
+::::::::
+
+This is not a resilient way to work on a research project. Many common events
+can cause big challenges:
+
+* someone leaves the project, temporarily or permanently
+* the project gets put on pause for a while
+* someone new wants to join the project and help out
+* someone wants to build new work on top of the project
+
+As you are left to puzzle everything out on your own, you daydream about a
+project where these events are much less disruptive. What might that look like?
+
+You'd want to be able to **collaborate** with other people starting early on in
+the project - it'd be nice if your other team members could review your work,
+provide feedback, and contribute to parts of the analysis. Then if you have to
+disappear for a while, other people can carry the project forward.
+
+You'd want to be **open and transparent** about the data - what the inputs are,
+what the outputs are, and how you got from A to B. When you're talking to a
+potential collaborator, a new teammate, or just getting back into the project
+after a break, being able to see the core data flows can save you a lot of
+pain.
+
+You'd also want the project outcomes to be **reproducible**, so researchers in
+other labs can build on the work that you're about to do, and confirm your
+results. Maybe you end up getting a new job, leaving someone else in the lab to
+keep the project going. In your new role, you might find that you yourself want
+to build on all your old work.
+
+We'll keep these principles in mind as we work through the material in this
+course.
+
+### Welcome
+
+TODO: break the fourth wall, get students to understand that they belong here
+
+### The research life cycle
+
+Research is cyclic by nature: we ask questions about the world, and as we investigate them, we make hypotheses, test those hypotheses, and reveal more questions along with our conclusions.
+You may have seen a diagram like this in a research methods class or as part of orientation for your research institute.
+If not, that's okay!
+You don't need to know it by heart to succeed in this course.
+
+[image]
+
+KM TODO: Paragraph: summary
+
+KM TODO: Paragraph: parts of the research process which benefit from collaboration
```

Comment:
(i'm not asking anyone else to imagine the diagram in my head; diagram finishing is friday's task)